audmetric/core/api.py

```diff
@@ -1229,13 +1226,9 @@ def word_error_rate(
 
 
 def _matching_scores(
-    truth: typing.Union[
-        typing.Union[bool, int], typing.Sequence[typing.Union[bool, int]]
-    ],
-    prediction: typing.Union[
-        typing.Union[bool, int, float], typing.Sequence[typing.Union[bool, int, float]]
-    ],
-) -> typing.Tuple[np.ndarray, np.ndarray]:
+    truth: (bool | int | Sequence[bool | int]),
+    prediction: (bool | int | float | Sequence[bool | int | float]),
```

Comment:
I guess here holds the same argument as with `linkability` and we should change it when tackling #61.

Reply:
The description of the follow up issue has been extended [here](https://github.com/audeering/audmetric/issues/61#:~:text=As%20mentioned%20below%2C%20the%20scope%20of%20the%20MR%20should%20be%20extended%3A)